app/mailers/discovered_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class DiscoveredMailer < ::ApplicationMailer
+  helper :discovered_hosts
+  def discovered_summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user = User.find(options[:user])) && user.mail_enabled?
+    set_url
+    @time = options[:time] || 'yesterday'
```

Comment:
Hmmm can we do better? There is no "now" when you read an old email, also yesterday and today are a bit fuzzy. Can we do: "Summary from 2015-11-26 16:10:30 UTC - 2015-11-27 16:10:30 UTC"? Or even better, use user's timezone instead.